.travis.yml

```diff
@@ -30,3 +30,23 @@ script:
   - cd $TRAVIS_BUILD_DIR/backend && npm test
   - cd $TRAVIS_BUILD_DIR/frontend && eslint .
   - cd $TRAVIS_BUILD_DIR/frontend && npm test
+
+before_deploy:
+  - sudo apt-get install python-dev -y
+  - cd $TRAVIS_BUILD_DIR/backend && pip install -r requirements.txt
+  - cd $TRAVIS_BUILD_DIR/frontend && npm install && cd $TRAVIS_BUILD_DIR
+  - openssl aes-256-cbc -K $encrypted_5d10786c9aae_key -iv $encrypted_5d10786c9aae_iv -in backend/devops/secret_files.tar.enc -out backend/devops/secret_files.tar -d
+  - cd $TRAVIS_BUILD_DIR/backend/devops && tar xvf secret_files.tar && cd $TRAVIS_BUILD_DIR
+
+deploy:
+  - provider: script
+    script: backend/bash/staging_quick_deploy.sh
```

Comment:
staging_quick_deploy also passes a tag --quick
which runs a subset of the ansible commands, which probably isn't what we want here

guessing that you made this quick_deploy to make it avoid running the tests?

If so my recommendation is to remove the tests from staging_deploy.sh and then have this call staging_deploy.sh